src/test/java/no/nav/veilarbregistrering/integrasjonstest/BrukerRegistreringServiceIntegrationTest.java

```diff
@@ -36,9 +35,10 @@
 import java.util.Optional;
 
 import static java.util.Optional.ofNullable;
+import static no.nav.fo.veilarbregistrering.bruker.FoedselsnummerTestdataBuilder.aremark;
 import static no.nav.fo.veilarbregistrering.profilering.ProfileringTestdataBuilder.lagProfilering;
 import static no.nav.fo.veilarbregistrering.registrering.bruker.OrdinaerBrukerRegistreringTestdataBuilder.gyldigBrukerRegistrering;
-import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
Vi bør forsøke å unngå bruk av Java6Assertions - den har ikke støtte for Optional m.m.